pyproject.toml

```diff
@@ -73,20 +73,11 @@ ignore = [
 
 [tool.ruff.lint.per-file-ignores]
 # necessary because of how file is included into documentation
-"examples/doc_examples/quickstart/test_my_design.py" = [
-    "E402",
-    "F811",
-]
+"examples/doc_examples/quickstart/test_my_design.py" = ["E402", "F811"]
```

Comment:
Now the below changes I'm not a fan of. Splitting these out onto multiple lines is nice to potentially add comments as to "why" it was set. Do you have a toml formatter in your editor?

Reply:
This is the "Even better TOML"-plugin for VS Code. 2.9M installs. I guess the rationale used is that if it fits on a single line, then use a single line. But also that it wouldn't do this if there were comments per line.

I read up a bit more and did now add formatting of `pyproject.toml` with a custom formatting that kept the previous indentation of subtables. The setting is also used for the VS Code plugin.

There is no setting for always breaking arrays into separate lines, but one can see from some other places in the file, like `extend-select` that it will keep separate lines if there are comments.